databases/week3/Functionality.sql

```diff
@@ -0,0 +1,41 @@
+USE meal_share_db;
+
+-- Get meals that has a price smaller than a specific price fx 90
+SELECT * FROM meal WHERE price < 90; 
+
+-- Get meals that still has available reservations
```

Comment:
So, this is the trap query. You need to sum the number_of_guests over all reservations for a given meal and then compare that number with the max_reservations of that meal.

There are at least two ways of doing this, both involve stuff that we didnt talk about in class:
 1. use the `COALESCE()` function: https://dev.mysql.com/doc/refman/8.0/en/comparison-operators.html#function_coalesce
 2. use `GROUP BY`, `SUM`,  and `HAVING: https://www.w3schools.com/sql/sql_having.asp

Reply:
Oh, and remember that there might be meals that don't have any reservations yet xD